core/src/saros/session/internal/SarosSession.java

```diff
@@ -164,6 +168,17 @@ public void send(List<User> recipients, IActivity activity) {
 
         @Override
         public void execute(IActivity activity) {
+          // Filters out resource activities for projects whose activity execution is disabled
+          if (activity instanceof IResourceActivity) {
+            SPath path = ((IResourceActivity) activity).getPath();
+
+            if (path != null && filteredProjects.contains(path.getProject())) {
```

Comment:
As far as I know there are EditorActivities with path = null to indicate an unknown editor.

Reply:
I only know of EditorActivities with `path=null` to indicate that no shared editor is selected/open.

But either way, if the path is `null`, we have no way of telling which project it belongs to, meaning we also can't apply it to a project. This would only be a problem if we have some activities that are applied to some "default project" such a case.

 But I will look through the `IResourceActivity` implementations to make sure.

Reply:
*Currently*, the only other resource activity besides `EditorActivity` that allows a `null` path is `JupiterActivity`, so it should not be a problem.

I am considering adding a comment to `AbstractResourceActivity` to avoid this being overlooked when adding new resource activities. But I am not sure whether it is really worth is.